### ISSUE TITLE
uri-parse and uri-port dont return same result

### ISSUE THREAD

**Issue by nightshade427** [fukamachi/quri#2]
If I do uri-parse vs uri-port I get diff results

``` lisp
(quri:uri-port (quri:uri "http://www.test.com"))                                                                                                                         
80                                                                                                                                                                                

(quri:parse-uri "http://www.test.com")                                                                                                                                   
"http"                                                                                                                                                                            
NIL                                                                                                                                                                               
"www.test.com"                                                                                                                                                                    
NIL                                                                                                                                                                               
NIL                                                                                                                                                                               
NIL                                                                                                                                                                               
NIL
```

Would it be preferable if uri-port returned a port if given like parse-uri does (NIL in this case)? So they are the same.

``` lisp
(quri:parse-uri "http://www.test.com:55")                                                                                                                                
"http"                                                                                                                                                                            
NIL                                                                                                                                                                               
"www.test.com"                                                                                                                                                                    
55                                                                                                                                                                                
NIL                                                                                                                                                                               
NIL                                                                                                                                                                               
NIL                                                                                                                                                                               

(quri:uri-port (quri:uri "http://www.test.com:55"))                                                                                                                      
55 
```

**Comment by Ambrevar** [comment]
@fukamachi @jathd (Second) thoughts on this?
I also find the inconsitency a bit problematic.  What about doing this the other way around, i.e. have `parse-uri` return the scheme default port?  Seems more logical to me.

**Comment by Ambrevar** [comment]
See #42 for my suggestion.

**Comment by Ambrevar** [comment]
Fixed with #42.
